Gemfile.lock

```diff
@@ -202,6 +204,3 @@ DEPENDENCIES
   unicorn
   will_paginate_mongoid (~> 2.0)
   yajl-ruby
-
-BUNDLED WITH
```

Comment:
I will remove it, this has prevented us from deploying the app in the past due to bundle version differences.